api/src/main/java/com/codeforcommunity/api/ITeamsProcessor.java

```diff
@@ -1,7 +1,16 @@
 package com.codeforcommunity.api;
 
 import com.codeforcommunity.auth.JWTData;
+import com.codeforcommunity.dto.team.*;
 
 public interface ITeamsProcessor {
   void disbandTeam(JWTData userData, int teamId);
+
+  void createTeam(JWTData userData, CreateTeamRequest request);
+
+  TeamDataResponse getTeam(JWTData userData, GetTeamRequest getTeamRequest);
```

Comment:
Same with deleteGoal, no JSON body for getTeam so just use a `Integer teamId` parameter instead of creating a DTO for it